.github/workflows/ci.yaml

```diff
@@ -0,0 +1,153 @@
+---
+name: ci
+on: [pull_request, push]
```

Comment:
I would suggest restricting this to branch master, otherwise the autoupdate pushes from upstream repos to their own branches will trigger a 'push' CI run unnecessarily.

Reply:
ok, fixed.